src/algebra/group_ring_action.lean

```diff
@@ -65,6 +65,14 @@ def mul_semiring_action.to_ring_equiv [mul_semiring_action G R] (x : G) : R ≃+
 { .. distrib_mul_action.to_add_equiv R x,
   .. mul_semiring_action.to_ring_hom G R x }
 
+/-- Compose a `mul_semiring_action` with a `monoid_hom`, with action `f r' • m`.
+See note [reducible non-instances]. -/
+@[reducible] def mul_semiring_action.comp_hom {N M : Type*} [monoid N] [monoid M] (f : N →* M)
```

Comment:
```suggestion
@[reducible] def mul_semiring_action.comp_hom (f : N →* M)
```
And put this in a section with `variables {M N}`.

Reply:
A nameless section you mean?

Reply:
Sure

Reply:
OK, done